maestro-client/src/main/java/maestro/drivers/AndroidDriver.kt

```diff
@@ -272,14 +272,14 @@ class AndroidDriver(
         when (swipeDirection) {
             SwipeDirection.UP -> {
                 val startX = deviceInfo.widthPixels / 2
-                val startY = deviceInfo.heightPixels
+                val startY = deviceInfo.heightPixels / 8
                 val endY = deviceInfo.heightPixels / 2
                 dadb.shell("input swipe $startX $startY $startX $endY $durationMs")
             }
             SwipeDirection.DOWN -> {
                 val startX = deviceInfo.widthPixels / 2
-                val startY = 0
-                val endY = deviceInfo.heightPixels / 2
+                val startY = deviceInfo.widthPixels / 2
```

Comment:
While you are on that change, could you change `widthPixels` and `heightPixels` to `widthGrid` and `heightGrid` (both from `deviceInfo` model). iOS actually expects us to send coordinates in points (which is what `grid` returns) whereas we are using pixels here